### DISCUSSION TITLE
How do I join to organization

### DISCUSSION THREAD

**Discussion by thuongtruong1009** [aio-libs/.github#20] [Q&A] [answered]
Hi there!
My name is Tran Nguyen Thuong Truong - @thuongtruong1009
Currently, I learning front-end.
I want to learn more and more about open-source code.
Hope to cooperate with you in the future.
How do I get an invitation to join organization!
Thank you!

**Comment by webknjaz** [comment, marked as answer]
Strictly speaking, you don't need an invitation to contribute. But there's https://github.com/aio-libs/triagers/issues/new/choose where you can ask to be in the Triage team. It could give you more privileges to change issue labels, but that's about it. I think that it's a good idea to start by just contributing something without that.
